core/templates/dev/head/css/oppia.css

```diff
@@ -2565,6 +2565,10 @@ div#ng-curtain {
   padding: 17px 12px 10px 12px;
   text-transform: uppercase;
 }
+.nav-block {
```

Comment:
Do you think we could give this class a name that's more "scoped" to the exploration editor? I think we should also file an issue to get rid of .nav in places other than the navbar, in the long term.